Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py

```diff
@@ -2602,6 +2614,63 @@ def remove_indicator_tag_command(client: Client, **kwargs) -> CommandResults:
     )
 
 
+def clone_ioc_command(client: Client, indicator_id: str = None) -> CommandResults:
+    """
+    Clone already imported indicator to be used with adding to a Trusted Circle
+    - Clone will fail with 403 if indicator value is not cloneable (403 not in ok_codes)
+    - ThreatStream - Error in API call 403 - {"message": "The entity <ip> is not cloneable"}
+    - You cannot clone
+        - type=string
+        - status=pending
+        - owner_organization_id=<your own org id>
+    Args:
+        indicator_id: Client to perform calls to Anomali ThreatStream service.
+    Returns:
+        (CommandResults).
+    """
+
+    res = client.http_request("POST", F"v2/intelligence/{indicator_id}/clone/", resp_type='response')
+
+    res_json = res.json()
```

Comment:
The default value for the `resp_type` is json, Try to remove the `resp_type=response` arg and the ` res_json = res.json()`

```suggestion
    res = client.http_request("POST", F"v2/intelligence/{indicator_id}/clone/")

```